14.04対応

http://jenkins.ros.org/job/ros-indigo-openhrp3_binarydeb_saucy_amd64/4/console
で以下のエラーになりました．これはどうやって直したらいいでしょうか？ openhrp3そのものからboost_system-mtをboost_systemにするとなにか問題ありましたっけ？あればあるで，こちら側の手元(https://github.com/tork-a/openhrp3-release )で治すことも可能です．

```
rpath,/usr/lib/atlas-base/atlas:/opt/ros/indigo/lib:/tmp/buildd/ros-indigo-openhrp3-3.1.7-2saucy-20141006-1155/obj-x86_64-linux-gnu/lib: 
/usr/bin/ld: cannot find -lboost_system-mt
/usr/bin/ld: cannot find -lboost_thread-mt
collect2: error: ld returned 1 exit status
make[4]: *** [bin/export-collada] Error 1
```

--- a/hrplib/hrpModel/CMakeLists.txt
+++ b/hrplib/hrpModel/CMakeLists.txt
@@ -60,7 +60,7 @@ if(UNIX)
       hrpCorbaStubSkel-${OPENHRP_LIBRARY_VERSION}
       hrpCollision-${OPENHRP_LIBRARY_VERSION}
       ${OMNIORB_LIBRARIES}
-      #boost_filesystem-mt boost_regex-mt
+      #boost_filesystem boost_regex
       ${Boost_REGEX_LIBRARY}
       #${Boost_LIBRARIES}
       dl)

--- a/hrplib/hrpUtil/CMakeLists.txt
+++ b/hrplib/hrpUtil/CMakeLists.txt
@@ -65,7 +65,7 @@ if(UNIX)
   if(NOT QNXNTO)
     target_link_libraries(${target}
       hrpCorbaStubSkel-${OPENHRP_LIBRARY_VERSION}
-      #boost_filesystem-mt boost_signals-mt
+      #boost_filesystem boost_signals
       ${Boost_FILESYSTEM_LIBRARY} ${Boost_SIGNALS_LIBRARY} ${Boost_SYSTEM_LIBRARY}
       #${Boost_LIBRARIES}
       ${PNG_LIBRARY} ${JPEG_LIBRARY} ${ZLIB_LIBRARY}
@@ -83,7 +83,7 @@ if(UNIX)
       )
   endif(NOT QNXNTO)
   if(APPLE)
-  target_link_libraries(${target} boost_system-mt)
+  target_link_libraries(${target} boost_system)
   endif() 
 elseif(WIN32)
   add_definitions(-DHRP_UTIL_MAKE_DLL)

--- a/server/ControllerBridge/CMakeLists.txt
+++ b/server/ControllerBridge/CMakeLists.txt
@@ -24,8 +24,8 @@ if(UNIX)
     hrpUtil-${OPENHRP_LIBRARY_VERSION}
     ${OPENRTM_LIBRARIES}
     ${Boost_PROGRAM_OPTIONS_LIBRARY} ${Boost_REGEX_LIBRARY}
-    #boost_filesystem-mt boost_program_options-mt boost_regex-mt
-    #boost_system-mt
+    #boost_filesystem boost_program_options boost_regex
+    #boost_system
     )
 elseif(WIN32)
   add_definitions(${OPENRTM_DEFINITIONS})

--- a/server/ModelLoader/CMakeLists.txt
+++ b/server/ModelLoader/CMakeLists.txt
@@ -35,7 +35,7 @@ if( COLLADA_DOM_FOUND )
       ${Boost_SIGNALS_LIBRARY}
       ${Boost_SYSTEM_LIBRARY} )
   else(WIN32)
-    set(extralibraries  ${COLLADA_DOM_LIBRARIES} boost_system-mt boost_thread-mt)
+    set(extralibraries  ${COLLADA_DOM_LIBRARIES} boost_system boost_thread)
   endif(WIN32)
 else( COLLADA_DOM_FOUND )
   set(extralibraries)

